### PR TITLE
OSDOCS-8900: adds images to embed apps tutorial

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-8900

Link to docs preview:
[Installation workflow review](https://68751--docspreview.netlify.app/microshift/latest/microshift_running_apps/microshift-embedding-apps-tutorial#microshift-installation-workflow-review_microshift-embedding-apps-tutorial)
[Embed applications workflow](https://68751--docspreview.netlify.app/microshift/latest/microshift_running_apps/microshift-embedding-apps-tutorial#microshift-embed-app-rpms-workflow_microshift-embedding-apps-tutorial)

QE review:
- N/A, images only; content already QE reviewed

### CODE REVIEW

**Review by DanielFroehlich — APPROVED** [review]

File: modules/microshift-embed-app-rpms-tutorial.adoc [hunk 1]
```diff
@@ -10,8 +10,16 @@ The following tutorial reviews the {microshift-short} installation steps and add
 
 [id="microshift-installation-workflow-review_{context}"]
 == Installation workflow review
-Embedding applications requires a similar workflow to embedding {microshift-short} into a {op-system-ostree} image. Reviewing those steps can help you understand the steps needed to embed an application:
-//larger concept image here
+Embedding applications requires a similar workflow to embedding {microshift-short} into a {op-system-ostree} image.
+
+* The following image shows how system artifacts such as RPMs, containers, and files are added to a blueprint and used by the image composer to create an ostree commit.
+* The ostree commit then can follow either the ISO path or the repo path to edge devices.
```
Comment (tmalove) [inline modules/microshift-embed-app-rpms-tutorial.adoc:16]:
```suggestion
* The ostree commit then can follow either the ISO path or the repository path to edge devices.
```
I noticed other instances in the section with the full spelling.

File: modules/microshift-embed-app-rpms-tutorial.adoc [hunk 2]
```diff
@@ -10,8 +10,16 @@ The following tutorial reviews the {microshift-short} installation steps and add
...
+* The ISO path can be used for disconnected environments, while the repo path is often used in places were the network is usually connected.
```
Comment (tmalove) [inline modules/microshift-embed-app-rpms-tutorial.adoc:17]:
```suggestion
* The ISO path can be used for disconnected environments, while the repository path is often used in places were the network is usually connected.
```

**Review by tmalove — APPROVED** [review]

**Pull request merged**